src/items/equipment/locus_blade_interference.json

```diff
@@ -0,0 +1,273 @@
+{
+  "_id": "09rWLl3PgofwjERQ",
+  "name": "Locus Blade, interference",
+  "type": "weapon",
+  "img": "icons/weapons/swords/sword-flanged-lightning.webp",
+  "system": {
+    "type": "",
+    "ability": "str",
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "eac",
+    "actionType": "mwak",
+    "activation": {
+      "type": "none",
+      "condition": "",
+      "cost": null
+    },
+    "ammunitionType": "charge",
+    "area": {
+      "effect": "",
+      "shape": "",
+      "units": "none",
+      "value": null
+    },
+    "attackBonus": 0,
+    "attributes": {
+      "ac": {
+        "value": ""
+      },
+      "customBuilt": false,
+      "dex": {
+        "mod": ""
+      },
+      "hardness": {
+        "value": ""
+      },
+      "hp": {
+        "max": "",
+        "value": 42
+      },
+      "size": "medium",
+      "sturdy": true
+    },
+    "attuned": false,
+    "bulk": "L",
+    "capacity": {
+      "max": 40,
+      "value": 0
+    },
+    "chatFlavor": "",
+    "container": {
+      "contents": [],
+      "includeContentsInWealthCalculation": true,
+      "isOpen": true,
+      "storage": [
+        {
+          "type": "slot",
+          "acceptsType": [
+            "fusion"
+          ],
+          "affectsEncumbrance": true,
+          "amount": 9,
+          "subtype": "fusion",
+          "weightProperty": "level"
+        },
+        {
+          "type": "slot",
+          "acceptsType": [
+            "ammunition"
+          ],
+          "affectsEncumbrance": true,
+          "amount": 2,
+          "subtype": "ammunitionSlot",
+          "weightMultiplier": 1,
+          "weightProperty": ""
+        }
+      ]
+    },
+    "critical": {
+      "effect": "Leech",
+      "parts": []
+    },
+    "damage": {
+      "parts": [
+        {
+          "name": "",
+          "formula": "2d8 + @abilities.str.mod",
+          "group": null,
+          "types": {
+            "acid": false,
+            "bludgeoning": false,
+            "cold": false,
+            "electricity": true,
+            "fire": false,
+            "healing": true,
+            "piercing": false,
+            "slashing": false,
+            "sonic": true
+          }
+        }
+      ]
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>The Locus blade forms a cutting beam of coherent energy. Like a Locus beam, the blade’s modulation can be reversed to provide healing. These blades produce a pleasing, low hum that grows louder and shriller the faster the wielder swings the weapon.</p>\n<p>A weapon fitted with a Locus crystal creates a unique vibratory radiation that disrupts the target—or, with a flip of a switch, heals their wounds. Locus weapons have the Healing weapon special property. After using this property five times, the weapon’s Locus crystal burns out and must be replaced.</p>"
```

Comment:
Do we need to add a `Healing` weapon property?

Reply:
I think the Healing weapon property is listed as type of damage.